.ci/pipelines/e2e-tests-gke-k8s-versions.Jenkinsfile

```diff
@@ -35,7 +35,7 @@ pipeline {
         }
         stage('Run tests for different k8s versions in GKE') {
             parallel {
-                stage("1.15") {
+                stage("1.17") {
```

Comment:
I think we should also update `runWith(lib, failedTests, '1.15', "eck-gke15-${BUILD_NUMBER}-e2e")`

Reply:
🤦 looks like I am still half asleep